### PR TITLE
Show tooltips based on Runescape's advanced settings

### LINKED ISSUE #1143: Advance Options (from in-game settings) Tooltips
If the option to have tooltips on certain options in advance options in game is turned off, runelites tooltips will not overlay on top of that. (aka so if i have prayer tooltip turned off in advanced options I dont get runelites tooltips if I have that setting turned on. 

Alternative have multiple tooltip options for different things. Couldn't tell if that part was posted before or not

### PR DESCRIPTION
**Pull request by jplsek**
If the advanced setting has the tooltip set to disabled, also disable RuneLite tooltips.

Closes #1143

~If preferred, I could also add a toggle to allow RuneLite tooltips anyways.~ Edit: Did that.

### CODE REVIEW

File: runelite-client/src/main/java/net/runelite/client/plugins/mousehighlight/MouseHighlightOverlay.java [hunk 1]
```diff
@@ -118,6 +120,19 @@ public Dimension render(Graphics2D graphics)
 			{
 				return null;
 			}
+
+			// If the tooltip is disabled in the advanced settings, don't show our tooltip
+			int tooltips = client.getSetting(Setting.TOOLTIPS);
+
+			if (groupId == WidgetID.PRAYER_GROUP_ID && (tooltips >> 30 & 1) == 1)
```
Comment (Adam-) [inline runelite-client/src/main/java/net/runelite/client/plugins/mousehighlight/MouseHighlightOverlay.java]: What do these magic constants mean? (bit 30 and 31)

**Comment by jplsek** [comment]
Alright, resolved conflicts.

I also added an option for this feature because I think that players would still like to have RuneLite's minimal tooltips to show up even when the more descriptive variant has been disabled.

**Comment by Nightfirecat** [comment]
Coming to this PR rather late, but why was this closed? Seems like this still resolves the issue in question.

**Comment by jplsek** [comment]
Not sure. I had another PR closed on the same day. But I didn't really care that much at the time.

**Comment by deathbeam** [comment]
No idea why I closed it really :man_shrugging:

**Comment by jplsek** [comment]
I'll just make new PRs for them soon.

**Pull request closed without merging**